src/rgw/support/rgw_support_bundle

```diff
@@ -128,14 +130,8 @@ class RGWSupportBundle:
         motr_trace_dir = os.path.join(log_dir, 'motr_trace_files')
         if os.path.exists(motr_trace_dir):
             # include the latest 5 log files of motr traces in support bundle
-            list_of_files = filter(lambda f: os.path.isfile(os.path.join(motr_trace_dir, f)),
-                            os.listdir(motr_trace_dir))
-            # sort the files based on last modification time
-            list_of_files = sorted(list_of_files,
-                            key = lambda f: os.path.getmtime(os.path.join(motr_trace_dir, f)),
-                            reverse=True)
-            list_of_files = list_of_files[0:5]
-            for file in list_of_files:
+            latest_5_files = RGWSupportBundle._get_latest_files(motr_trace_dir)
```

Comment:
just say latest_files , let five latest files remain in comment, as of line 132

Reply:
done